crates/layout/src/proto/layout_node.rs

```diff
@@ -12,12 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::layout_node::{self};
+use dc_proto::{
```

Comment:
I'm confused because the crate `dc_proto` doesn't have much in it -- like it doesn't have any of the dependencies below in it. Can you explain somewhere how this all works?